packages/office-ui-fabric-react/src/components/Callout/CalloutContent.tsx

```diff
@@ -136,20 +137,22 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
 
     let contentMaxHeight: number = this._getMaxHeight() + this.state.heightOffset!;
     let beakVisible = isBeakVisible && (!!targetElement || !!target);
+
     let content = (
       <div
         ref={ this._resolveRef('_hostElement') }
         className={ css('ms-Callout-container', styles.container) }
       >
         <div
           className={
-            css(
+            mergeStyles(
               'ms-Callout',
               styles.root,
               className,
-              directionalClassName
-            ) }
-          style={ positions ? positions.calloutPosition : OFF_SCREEN_STYLE }
+              directionalClassName,
+              positions ? positions.calloutPosition : OFF_SCREEN_STYLE,
```

Comment:
Since positions can change a bunch, I think it should stay as an inline style so it doesn't get reregistered if it's moving about or resizing.